docs/layouts/partials/examples/relationship.html

```diff
@@ -0,0 +1,9 @@
+<!-- Creates an example relationships block from the registry's example naming in `.docs` -->
+<!-- Note: this will only replace the endpoint and service type for demonstration purposes. -->
+<!-- To use this partial, pass the image data as the context. -->
+<!-- Ex: partial "examples/relationship" $data  -->
```

Comment:
1) Will these comments show up in the output?  Doesn't the template syntax have its own comment syntax to keep that from happening?
2) Why 4 comments instead of one multi-line comment?

(Applies to other files, too.)

Reply:
1. They won't show up in the output. The only thing that will is what's passed to return.

Reply:
2. because. I fix